tests/coreml/test_AllNeuralNetworkConverters.py

```diff
@@ -1,8 +1,11 @@
+from distutils.version import StrictVersion
```

Comment:
Please move this to line 7, right below `from coremltools.proto.FeatureTypes_pb2 import ImageFeatureType`. Thanks.

Reply:
Will do. One question, do you have a reference for the order I should follow? You've asked me to move a few and I'm doing it blindly, but I'm a bit lost on the reasoning behind it. I'm following the [PEP 8 recommended ordering](https://www.python.org/dev/peps/pep-0008/#imports):

```
Imports should be grouped in the following order:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.
```

Which would place distutils (standard library) above coremltools (third party) instead of the other way around.

Reply:
My feeling is that sorting them by alphabet makes a cleaner view of imported packages. It's a simple and easy-to-be-remembered rule to me. :)